infrastructure-setup/scripts/prepare-jenkins-builds.sh

```diff
@@ -21,7 +21,7 @@ oc start-build -n cd jenkins-master --follow
 oc start-build -n cd jenkins-slave-base --follow
 oc start-build -n cd jenkins-webhook-proxy --follow
 
-cd ${BASE_DIR}/ods-project-quickstarters/jenkins-slaves/maven/ocp-config
+cd ${BASE_DIR}/ods-quickstarters/common/jenkins-slaves/maven/ocp-config
 yes 'y' | tailor update -v --force
 
-oc start-build -n cd jenkins-slave-maven --follow
+oc start-build -n cd jenkins-slave-maven --follow
```

Comment:
Why is the maven slave here at all? I'd assume we can remove it here. It is enough to have it listed in infrastructure-setup/scripts/prepare-jenkins-slave-builds.sh.

Reply:
@michaelsauter thank you for your comment. I have implemented it.